ios/RNSScreenStackHeaderConfig.h

```diff
@@ -1,13 +1,35 @@
-#import <React/RCTConvert.h>
+#ifdef RN_FABRIC_ENABLED
+#import <React/RCTViewComponentView.h>
+#import <UIKit/UIKit.h>
+#import "RNSScreenStackHeaderSubviewComponentView.h"
+#else
 #import <React/RCTViewManager.h>
+// TODO: move this import when SearchBar is implemented on Fabric
+#import "RNSSearchBar.h"
+#endif
 
+#import <React/RCTConvert.h>
 #import "RNSScreen.h"
-#import "RNSSearchBar.h"
 
-@interface RNSScreenStackHeaderConfig : UIView
+@interface RNSScreenStackHeaderConfig :
+#ifdef RN_FABRIC_ENABLED
```

Comment:
Should we use such convention or the one with `RNS_BASE_VIEW `?

Reply:
I believe it's clearer this way:

[fd57615](https://github.com/software-mansion/react-native-screens/pull/1417/commits/fd57615b382052f5b82b1cf2813509a95268076b)